x-pack/plugins/security_solution/public/common/components/drag_and_drop/draggable_wrapper_hover_content.tsx

```diff
@@ -5,397 +5,9 @@
  * 2.0.
  */
 
-import {
-  EuiButtonIcon,
-  EuiFocusTrap,
-  EuiPanel,
-  EuiScreenReaderOnly,
-  EuiToolTip,
-} from '@elastic/eui';
+import React, { useEffect, useState } from 'react';
```

Comment:
We can rename this file now to like `use_get_timeline_id_from_dom` or something like that. I did it in a different PR, but felt out of scope